packages/Webkul/Installer/src/Console/Commands/DefaultUser.php

```diff
@@ -452,4 +362,118 @@ public function getTimeZones(): array
 
         return $formattedTimezones;
     }
+
+    protected function generateWarnings($message)
```

Comment:
missing parameter types, return types and docblock for all new functions.

Reply:
added parameter types, return types and docblock